### DISCUSSION TITLE
Nullable types for computed columns?

### DISCUSSION THREAD

**Discussion by gabrielhora** [sqlc-dev/sqlc#1030] [Q&A]
Hi there,

I'm trying to come up with a solution for having nullable types for computed columns.
I know I can override table column types via `overrides` in sqlc.yaml, is there a way to do the same for queries?

This is a simple example, `min` and `max` may return null. How can I deal with this in sqlc?

```go
const test = `-- name: Test :one
select min(age)::int as min_age, max(age)::int as max_age from my_table
`

type TestRow struct {
	MinAge int32
	MaxAge int32
}

func (q *Queries) Test(ctx context.Context) (TestRow, error) {
	row := q.db.QueryRowContext(ctx, test)
	var i TestRow
	err := row.Scan(&i.MinAge, &i.MaxAge)
	return i, err
}
```

Thanks a lot for your time.

**Comment by nmsobri** [comment]
no response? may i know why?
